docs/Process.md

```diff
@@ -31,15 +29,25 @@ further tweaked to align with the HLSL team's goals and priorities.
 ## Proposing a Feature
 
 By far the best way for an external contributor to propose a feature is through
-GitHub issues (See the section below on "Filing Issues"). If you can't be
-deterred from writing a proposal yourself you must find a member of the HLSL
-team to act as a _Sponsor_ for the change. The _Sponsor_ is responsible for
-tracking and helping change proposals through the proposal life cycle.
+GitHub issues (See the section below on "Filing Issues"). Issues in this
+repository will be used to publicly track feature requests for the HLSL language
+and HLSL runtime interfaces. Direct tooling feature requests to the
+[DirectXShaderCompiler](https://github.com/microsoft/DirectXShaderCompiler/issues/new).
+
+If you write a proposal yourself you must find a member of the HLSL team to act
+as a _Sponsor_ for the change. The _Sponsor_ is responsible for tracking and
+helping change proposals through the proposal life cycle. For Vulkan-specific
+features we require at least one _Sponsor_ from the HLSL team and one _Sponsor_
+from the Khronos Vulkan Working Group. If you need assistance finding a
+_Sponsor_ for a proposal reach out to the [HLSL Team](Contact.md).

```

Comment:
That won't help to find a sponsor from Khronos. I don't know how that would be accomplished really.

Reply:
The point of that is to reach out to us and we can help find someone (likely going through the Google team).